pom.xml

```diff
@@ -65,9 +65,8 @@
   </distributionManagement>
 
   <properties>
-    <mavenVersion>3.0</mavenVersion>
-    <maven.compiler.source>1.7</maven.compiler.source>
-    <maven.compiler.target>1.7</maven.compiler.target>
+    <mavenVersion>3.1.0</mavenVersion>
+    <maven.compiler.release>8</maven.compiler.release>
```

Comment:
defining `maven.compiler.release` will make it incompatible with Java 8.
Please use `maven.compiler.source` and `maven.compiler.target` for now. Also see my other comment.

`jlink` will be obtained from the `$PATH` or `toolchains.xml`, so this does not prevent this plugin from running with Java 8.